app/views/nic/_base_form.html.erb

```diff
@@ -63,17 +63,17 @@
 
 <% if SETTINGS[:unattended] %>
   <%= checkbox_f f, :managed,
-                 :label_help => _("Should this interface be managed via DHCP and DNS smart proxy and should it be configured during provisioning?"),
+                 :label_help => _("If managed flag is enabled, external services such as DHCP, DNS and TFTP will be configured according to the provided information.<br/>What services will be updated is determined by the subnet and domain that are selected for this interface. If you wish to manage<br/> only subset of these services, change the subnet and domain proxies configuration. When proxy is set to 'None', the corresponding<br/> service configuration will be skipped.<br/><br/>If manage flag is disabled, none of the service will be configured for this interface, even when it's configured for subnet and domain.<br/><br/>The flag also determines whether the interface should be configured during host provisioning, so that the configuration remains after reboot."),
```

Comment:
> If you wish to manage only subset of these services, change the subnet and domain proxies configuration.

It's a bit unclear where to change this configuration, it would be awesome if you provided a link to those settings directly.